core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -238,6 +239,42 @@ describe('Contributions and review component', function() {
       $scope.$apply();
     });
 
+    it('should remove resolved suggestions when suggestion ' +
+      'modal is opened and remove button is clicked', fakeAsync(function() {
+      spyOn($uibModal, 'open').and.returnValue({
+        result: Promise.resolve(['id1', 'id2'])
+      });
+      let removeSpy = spyOn(
```

Comment:
Prefer `const` if the variable is not being reassigned. Here and below.

Reply:
done.